tests/conftest.py

```diff
@@ -173,8 +173,8 @@ def catalog(id_generator, metastore, warehouse):
 
 
 @pytest.fixture
-def test_session(catalog):
-    with Session("TestSession", catalog=catalog) as session:
+def test_session(cloud_test_catalog):
```

Comment:
why is this change?

Reply:
We have `cloud_test_catalog` thing which has `catalog` inside that has set up for all mocked clouds to return data from `tests/data.py`.  We are using it extensively across the tests to be able to test with all the clouds.

In one of the tests I needed `test_session`  but also this special `catalog` from `cloud_test_catalog` to test for all the clouds, so instead of creating duplicate `test_session` I just used this catalog instance inside existing one. I think it's convenient to have this one here regardless.

Reply:
`cloud_test_catalog` does it mean that we run some extra code, initialization and cleanup for more tests where we don't need to?

Reply:
It just does simple setup, but the bigger problem is that enforces running tests for every cloud type and cloud versioning which means it will slow tests significantly where it doesn't need to (e.g unit tests where we manually create local dirs / files and test on them) so I'm reverting this change and adding `session` property on `CloudTestCatalog` class to be used where we need that mocked clouds and running tests for each of those (usually func tests)